components/SectionPage.vue

```diff
@@ -0,0 +1,328 @@
+                    :value="getCommentCountById(story.legacyId, moreResultsDisqusData)"
+                    :href="`/${story.ancestry[0].slug}/${story.meta.slug}?to=comments`"
+                  />
+                </template>
+              </article-metadata>
+            </v-card>
+          </div>
+        </div>
+        <v-spacer size="double" />
+        <div
+          v-if="moreResults && moreResults.length > 0"
+          class="l-container u-align-center"
+        >
+          <v-button
+            class="more-results"
+            :disabled="!moreResultsLoaded"
+            @click="getMoreResults"
+          >
+            <span v-if="moreResultsLoaded">More Results</span>
+            <span v-if="!moreResultsLoaded">Loading...</span>
+          </v-button>
+        </div>
+      </section>
+      <v-spacer />
+    </div>
+  </div>
+</template>
+
+<script>
+import gtm from '@/mixins/gtm'
+import disqus from '@/mixins/disqus'
+import { mapState } from 'vuex'
+
+const {
+  formatTags,
+  fuzzyDateTime,
+  getArticleImage,
+  hasGallery
+} = require('~/mixins/helpers')
+
+export default {
+  name: 'Section',
+  components: {
+    ArticleMetadata: () => import('nypr-design-system-vue/src/components/ArticleMetadata'),
+    LoadingIcon: () => import('nypr-design-system-vue/src/components/animations/LoadingIcon'),
+    VButton: () => import('nypr-design-system-vue/src/components/VButton'),
+    VCard: () => import('nypr-design-system-vue/src/components/VCard'),
+    VCounter: () => import('nypr-design-system-vue/src/components/VCounter'),
+    VSpacer: () => import('nypr-design-system-vue/src/components/VSpacer')
+  },
+  mixins: [gtm, disqus],
+  props: {
+    name: {
+      type: String,
+      default: null
+    },
+    descendantOf: {
+      type: Number,
+      default: null,
+      required: true
+    }
+  },
+  async fetch () {
+    // get featured stories
+    await this.$axios
+      .get('pages/?type=news.ArticlePage&fields=*&order=-publication_date&show_on_index_listing=true&descendant_of=' + this.descendantOf + '&limit=2&show_as_feature=true')
+      .then((response) => {
+        this.featuredStories = response.data.items
+        response.data.items.forEach((item) => {
+          this.featuredStoriesDisqusThreadIds.push(item.legacyId)
+        })
+        this.featuredStoriesLoaded = true
+      })
+  },
+  data () {
+    return {
+      featuredStories: null,
+      featuredStoriesDisqusThreadIds: [],
+      featuredStoriesDisqusData: null,
+      featuredStoriesLoaded: false,
+      moreResults: [],
+      moreResultsDisqusThreadIds: [],
+      moreResultsDisqusData: null,
+      moreResultsLoaded: false,
+      moreResultsOffset: 0
+    }
+  },
+  computed: {
+    ...mapState('global', {
+      defaultImage: state => state.defaultImage,
+      defaultImageArts: state => state.defaultImageArts,
+      defaultImageFood: state => state.defaultImageFood,
+      defaultImageNews: state => state.defaultImageNews
```

Comment:
Another thing we don't need to do immediately, but wondering if it makes sense to move some of these static default settings out of global state and into `publicRuntimeConfig` https://nuxtjs.org/docs/2.x/directory-structure/nuxt-config/#using-your-config-values

Reply:
We could - I was just thinking it's easier for future devs to find things if they are all in one place